From d9dd5712d6635ff05b13c96ae2c94e5d68474b89 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 27 Jun 2022 18:45:02 +0200
Subject: [PATCH 172/417] arm64: dts: rockchip: rk3399-pinebook-pro: Support
 both Type-C plug orientations

This fixes USB 3.0 and Alt-DP modes in reverse plug orientation.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 03c53510c..9bacd625f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -1128,6 +1128,7 @@ &usbdrd3_0 {
 &usbdrd_dwc3_0 {
 	dr_mode = "otg";
 	extcon = <&typec_extcon_bridge>;
+	snps,usb3-phy-reset-quirk;
 	status = "okay";
 };
 
-- 
2.35.3

